src/schema/frugalos.rs

```diff
@@ -229,6 +230,27 @@ pub struct SegmentRequest {
     pub segment: u16,
 }
 
+/// デバイス単位でのRPC要求。
+#[allow(missing_docs)]
+#[derive(Debug, Serialize, Deserialize)]
+pub struct DeviceRequest {
+    pub device_id: DeviceId,
+}
+
+/// Inspect Physical Device
+#[derive(Debug)]
+pub struct InspectPhysicalDeviceRpc;
+impl Call for InspectPhysicalDeviceRpc {
+    const ID: ProcedureId = ProcedureId(0x000b_0001);
+    const NAME: &'static str = "frugalos.device.inspect_physical_device";
+    type Req = DeviceRequest;
+    type ReqDecoder = BincodeDecoder<Self::Req>;
+    type ReqEncoder = BincodeEncoder<Self::Req>;
+    type Res = Result<PhysicalDeviceInspection>;
+    type ResDecoder = BincodeDecoder<Self::Res>;
+    type ResEncoder = BincodeEncoder<Self::Res>;
```

Comment:
`BincodeEncoder` and `BincodeDecoder` work in monolithic (i.e., will block the current thread). And there is a possibility that the response size of this RPC becomes huge. Thus it may be worth to enable asynchronous response for this RPC by implementing [enable_async_response](https://docs.rs/fibers_rpc/0.2.17/fibers_rpc/trait.Call.html#method.enable_async_response) method.

Reply:
Thank you for reviewing. I'll check it.